src/Core/Usecase/Post/Export.php

```diff
@@ -223,7 +225,23 @@ public function constructFilters($filters, $job_filters = null)
      */
     public function constructSearchData($job)
     {
+        $form_ids_by_attributes = $this->formAttributeRepository->getFormsByAttributes($job->fields);
         $filters = $this->constructFilters($this->filters, $job->filters);
+
+        if (!empty($form_ids_by_attributes)) {
+            $filters['form'] = array_unique(
+                array_merge(
+                    isset($filters['form']) ? $filters['form'] : [],
+                    array_map(
+                        function ($itm) {
```

Comment:
This is stupid but could you call the var $item?

Reply:
yea that makes more sense. Thanks